vignettes/efficient-calculations.Rmd

```diff
@@ -100,64 +100,81 @@ results <- runSimulations(simulations = list(sim100, sim200, sim300, sim400, sim
 # Results in now a list of SimulationResults
 ```
 
-This implementation is also fairly straight forward. We load the simulation multiple times from the same file (note the usage of `loadFromCache=FALSE` to ensure that we always get a new instance of a simulation and not the same instance) and we set the dose. Then, we call the `runSimulations` function with the list of simulations. The engine will run these simulations in parallel, which means that the overall execution time should be slightly more than 1h, as each run would be executed **at the same time**.
+This implementation is also fairly straight forward. We load the simulation multiple times from the same file (note the usage of `loadFromCache = FALSE` to ensure that we always get a new instance of a simulation and not the same instance) and we set the dose. Then, we call the `runSimulations` function with the list of simulations. The engine will run these simulations in parallel, which means that the overall execution time should be slightly more than 1h, as each run would be executed **at the same time**.
 
 
-## Running one or more simulations multiple times in a loop

```

Comment:
I have the feeling that this was not the right caption? I did not see a section where you describe a loop. I propose a new caption.

Reply:
I would only use batch if it  was in a loop such as an algorithm (PI, sensitivity etc..) .For a finite number of run, I am not sure that this is really required.....